elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -303,27 +267,9 @@ export abstract class BaseTransaction {
 			);
 		}
 
-		const feeError = this.validateFee();
-
-		if (feeError) {
-			errors.push(feeError);
-		}
-
 		return createResponse(this.id, errors);
 	}
 
-	public validateFee(): TransactionError | undefined {
```

Comment:
How fees are validate now?

Reply:
Isn't fee part of the signature? 🤔

Reply:
fee is currently not part of the signature